junit5/src/main/java/org/jboss/weld/junit5/WeldInitiator.java

```diff
@@ -33,21 +33,22 @@
 /**
  * JUnit 5 initiator - can be used to customize the Weld SE container started by {@link WeldJunit5Extension}.
  *
- * <pre>
- * &#64;ExtendWith(WeldJunit5Extension.class)
- * public class SimpleTest {
+ * <p><b>Example:</b>
+ * <pre>{@code
```

Comment:
Hm, what's the point of using `{@code}` inside the `<pre>` tag? I think that it does not improve the readability of the  code snippet, quite the opposite...

Reply:
Actually I do not fully remember, but that is what I ended up using in my projects to get the best result in the generated JavaDocs.

Reply:
Trying it locally, the previous `&#64;Test` will give you simply `@Test` in javadoc.
The new `&#64;{@code Test` will result in something like `@ Test` with a whitespace in there.

Personally, I find the older format more readable as well.

Reply:
Something with it actually being marked as code in the generated HTML which makes it better for screen readers and makes it possible to do syntax highlighting and so on or something like that.

Reply:
Alternatively you could generate the JavaDoc with Java 18 and use the new snippet support where you can have the example in an actual file and just included it into the JavaDoc, much like with AsciiDoc. Then you can even be sure the snippets compile and are correct when taking them from actual test code. :-)

Reply:
Generating HTML javadoc, I see no difference at all (JDK 11) and from within IDE I see what I described above which is weird

Reply:
The space when displayed in IntelliJ is actually an IntelliJ bug: https://youtrack.jetbrains.com/issue/IDEA-216423

The generated HTML **is** differently, one is just a preformatted text, one is properly marked as code.

In a standard browser without plugins this indeed does not make a difference.

But as I said earlier, it is then better for accessibility, for example for screen readers and if you for example have a Browser plugin that adds code highlighting to text marked as code, you will get nice highlighting.

But as I said in earlier PRs, it is your project and my PRs are just kind suggestions, if you don't like it, I'll remove it. :-)